src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1994,6 +1994,18 @@ internal enum ErrorCode
 
         #endregion
 
+        #region diagnostics introduced for C# 11.0
+
+        ERR_Raw_string_literals_are_not_allowed_in_preprocessor_directives = 9100,
```

Comment:
Please leave a PROTOTYPE marker to re-number

Reply:
what's a "PROTOTYPE marker"?

Reply:
It's like a TODO marker, but with "PROTOTYPE". For example: `// PROTOTYPE: renumber error codes before merging feature`
The `main` branch checks that the upper-case word "PROTOTYPE" can't be merged. We use such markers to track TODOs in feature branches.

Reply:
hrmm... maybe i misunderstood though.  Why would we renumber these?  Placing these in their own group means that we can easily add/refine this in future without colliding with other error codes for other feature segments.

Reply:
Own group is fine. We just don't know where the group will start by the time we merge.

Reply:
Gentle ping

Reply:
done!